Assets/Scripts/UI/Interaction Wheel/WheelButton.cs

```diff
@@ -0,0 +1,50 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+using System.Collections;
+
+
+public class WheelButton : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler {
+
+    public Image circle;
+	public Image icon;
+	public InteractableWheel myMenu;
+	public float speed = 8f;
+    public Interactable myAction;
+
+    public Color baseColor = Color.white;
+    public Color hoverColor = new Color(0.9f, 1.0f, 0.1f);
+
+	public void Anim(){
+        icon.sprite = myAction.icon;
+		StartCoroutine(AnimateButtonIn());
+	}
+
+	//make button grow UwU
```

Comment:
Comment. Method name should make clear what it does, perhaps something like AnimateGrowButton, or just GrowButton